rockcraft.yaml

```diff
@@ -1,7 +1,7 @@
 name: glauth
 base: bare
-build-base: ubuntu:22.04
-version: "2.2.1"
+build-base: ubuntu@22.04
+version: "2.3.0"
```

Comment:
The 56ccb88 commit is built on top of changes from v2.3.0, so perhaps we should name it differently

Reply:
I am confused by the upstream. It seems that this commit is supposed to be on top of v2.3.1, but the upstream seems failed to release v2.3.1?

Reply:
There is no v2.3.1 yet. Once it's published, we can change the tag and source to follow the upstream.